src/bio/bio_wal.c

```diff
@@ -1039,6 +1039,11 @@ bio_wal_commit(struct bio_meta_context *mc, struct umem_wal_tx *tx, struct bio_d
 	free_data_csum(&dc_arr);
 	if (biod != NULL)
 		bio_iod_free(biod);
+
+	if (biod_data != NULL && biod_data->bd_async_post == 1 &&
+	    biod_data->bd_post_latency != NULL)
+		*biod_data->bd_post_latency = daos_get_ntime() - *biod_data->bd_post_latency;
```

Comment:
This change will make the "bio_post latency" overlapping with "vos_update" latency, I'm wondering if it'll make things worse?

These "vos_update latency" & "bio_post latency" are for developers only, and it's expected that in md-on-ssd mode, "bio_post latency" is zero and "vos_update latency" includes "WAL & async post commit latency", I think we'd keep it as-is.

Reply:
OK from my side, I am going to set this PR back to draft.
I will cancel it or not, according to the Jira ticket [DAOS-14099](https://daosio.atlassian.net/browse/DAOS-14099?focusedCommentId=127854) discussion.